plugins/chat/assets/javascripts/discourse/routes/chat.js

```diff
@@ -39,6 +39,8 @@ export default class ChatRoute extends DiscourseRoute {
       "chat.channel.index",
       "chat.channel.near-message",
       "chat.channel-legacy",
+      "chat.channel.info.settings",
```

Comment:
I wonder if we can get away with using the routes defined in `chat-drawer-router.js` instead of duplicating the list here as well? 🤔

Reply:
Yep, this is a good point. I think for now we need this as it is used to intercept the routes and abort the transition when drawer mode is preferred (otherwise it will load in fullscreen). I will possibly follow up with a separate PR to simplify this so that we don't have to add route names in 2 locations for drawer in future.